cpp/src/io/csv/reader_impl.cu

```diff
@@ -30,7 +30,7 @@
 #include "legacy/datetime_parser.cuh"
 #include "legacy/type_conversion.cuh"
 
-#include <utilities/cudf_utils.h>
+#include <utilities/legacy/cudf_utils.h>
```

Comment:
I checked and nothing in this file uses anything in `cudf_utils.h`. Please remove this include.

Reply:
Yeah, I believe we can remove this header.